README.md

```diff
@@ -47,6 +47,13 @@ The AWS for Fluent Bit image includes the following contents:
 
 A change in any one of these pieces would lead to a change in our version number.
 
+**Are there edge cases to the rules on breaking backwards compatibility?**
```

Comment:
If anyone disagrees with adding these notes, let's discuss. We could remove them. We definitely need to be more clear that there are ways in which we do sometimes break backwards compatibiity. One  other option is just to be more vague and not ot make any guarantee.